src/Blake.BuildTools/Blake.BuildTools.csproj

```diff
@@ -47,8 +47,8 @@
 
     <ItemGroup>
       <PackageReference Include="YamlDotNet" Version="16.3.0" />
-	  <PackageReference Include="Microsoft.AspNetCore.Components" Version="9.0.0" />
-	  <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="9.0.0" />
+	  <PackageReference Include="Microsoft.AspNetCore.Components" Version="8.0.0" />
```

Comment:
Same here - please keep the package versions in tact, don't roll back to earlier versions.